fix: support TextInput editable={false} prop check in isEnabled/isDisabled matchers (#135)
testing-library/jest-native: README.md

Check whether or not an element is disabled from a user perspective.

> This matcher will check if the element or its parent has any of thoses props value:

Reviewer: ```suggestion
This matcher will check if the element or its parent has any of those props:
```

- `disabled={true}`
- `accessibilityState={{disabled: true}}`

> - `editable={false}` for `TextInput` only

Reviewer 2: ```suggestion
- `editable={false}` (for `TextInput` only)
```

> This matcher will check if the element or its parent has any of those props :
> 
> - `disabled={true}`
> - `accessibilityState={{disabled: true}}`
> - `editable={false}` (for `TextInput` only)

Reviewer 2: ```suggestion
This matcher will check if the element or its parent has any of the following props :
- `disabled`
- `accessibilityState={{ disabled: true }}`
- `editable={false}` (for `TextInput` only)
```
